content_script.js

```diff
@@ -14,8 +14,7 @@ class HTMLSerializer {
      *     ignored while serializing a document.
      * @const
      */
-     // TODO(sfine): Process links.
-    this.FILTERED_TAGS = new Set(['script', 'noscript', 'style', 'link']);
+    this.FILTERED_TAGS = new Set(['script', 'noscript', 'style']);
```

Comment:
Yes, remove it from this pull request. That makes the refactoring review easier.